_data/simple-icons.json

```diff
@@ -7059,6 +7059,11 @@
             "hex": "00AFAA",
             "source": "https://www.here.com/company/media-assets"
         },
+        {
+            "title": "Hermes",
+            "hex": "0091CD",
+            "source": "https://www.myhermes.de"
```

Comment:
I think we should maybe link to https://www.hermesworld.com/? Seems to be their international site, and is what is linked to on https://en.wikipedia.org/wiki/Hermes_Europe.